### PR TITLE
Added external toggle to markdown mode

### LINKED ISSUE #144: Add external toggle to markdown mode
- [x] Remove the `TabSwitcher` component from markdown mode.
- [x] Add ability to toggle markdown mode externally.

@AbhayVAshokan _a

### PR DESCRIPTION
**Pull request by AbhayVAshokan**
Fixes #144 
- `markdownMode` prop can now be used to toggle between markdown mode and rich text mode using an external state.
- Removed TabSwitcher component from markdown mode.
- Passed the `onChange` and `onSubmit` methods to markdown mode: This will make sure that if the text content is submitted when the markdown mode is active, the most recent changes are obtained in the HTML format.
- Updated documentation.

@labeebklatif _a please review.

cc: @karthiknmenon

### CODE REVIEW

File: example/App.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import React, { useRef } from "react";
+import React, { useRef, useState } from "react";
+import { Down } from "@bigbinary/neeto-icons";
```
Comment (labeebklatif) [inline example/App.js:2]: let's use aliased import for neeto-icons

File: example/App.js [hunk 2]
```diff
@@ -7,6 +8,7 @@ import HighlightText from "./components/HighlightText";
 import ListItems from "./components/ListItems";
 
 import { Editor } from "../lib";
+import Dropdown from "../lib/components/Common/Dropdown";
```
Comment (labeebklatif) [inline example/App.js:11]: can we use alias for components. let's create one if not already exists

File: lib/components/Editor/CustomExtensions/Markdown/index.js [hunk 3]
```diff
@@ -36,6 +44,15 @@ const MarkdownEditor = ({
 
   const handleChange = (e) => {
     setContent(e.target.value);
+    onChange(e.target.value);
+  };
+
+  // Submit on Ctrl+Enter and Cmd+Enter
+  const handleSubmit = (e) => {
+    if (e.keyCode === 13 && (e.metaKey || e.ctrlKey)) {
```
Comment (labeebklatif) [inline lib/components/Editor/CustomExtensions/Markdown/index.js:52]: can we move this 13 to a constant like 'KEYBOARD_KEY_CODE_ENTER' or something like that?

**Review by labeebklatif — COMMENTED** [review]
@AbhayVAshokan _a few minor suggestions. Also, please have a look at this [issue](https://www.loom.com/share/30ca5c6f136b49beaad836c65401e1e5)

**Comment by AbhayVAshokan** [comment]
@labeebklatif _a please review the changes.
1. Please check the new approach for describing `onKeyDown` events. This approach seems more self-explanatory than moving them to a different constants file.
2. Solved bubble menu issue mentioned in the video. Could not find a better approach than this.

**Review by labeebklatif — APPROVED** [review]
LGTM

**Pull request merged**